### PR TITLE
css notebook

### PR DESCRIPTION
**Pull request by mujeeb100**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  width: 90%;
+  max-width: 800px;
+  min-width: 400px;
+  height: 480px;
+  margin: 0 auto;
+  background: #fafafa;
+  box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.3);
+  border-radius: 10px;
+  overflow: hidden;
+}
```
Comment (Rehan-Khan-Engineer) [inline style.css:29]: You can group similar properties in the entire stylesheet @mujeeb100

**Comment by Rehmatulla-Khan** [comment]
@mujeeb100 grp the css properties

**Review by Tanvir04khan — APPROVED** [review]
Group similar properties

**Review by syedabdullahali — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (mujeeb100) [reply style.css:29]: how to group similar properties ??

**Pull request merged**